README.md

```diff
@@ -25,13 +25,52 @@ And 4.2.14+amd64 would only apply to the amd64 release image.
 ### Add Releases To Channels
 
 Edit the appropriate file in `channels/`.
-For example, to add a release to stable-4.2 you would edit `channels/stable-4.2.yaml`.
-Channel semantics are documented [here][channel-semantics].
+For example, to add a release to candidate-4.2 you would edit [`channels/candidate-4.2.yaml`](channels/candidate-4.2.yaml).
 
 The file contains a list of versions.
 Please keep the versions in order.
 And LEAVE COMMENTS if you skip a version.
 
+#### Feeder Channels
+
+Channel semantics, as documented [here][channel-semantics], show nodes and edges being promoted to successive channels as they prove their stability.
+For example, a 4.2.z release will appear in `candidate-4.2` first.
+Upon proving itself sufficiently stable in the candidate channel, it will be promoted into `fast-4.2`.
+Some time after landing in `fast-4.2`, it will appear in `stable-4.2`.
+
+_Note:_ Once we have phased release rollouts, we will drop the fast/stable distinction from this repository and promote to a unified fast/stable channel with a start time and rollout duration.
+Until then, we are using fast channels to feed stable channels with a delay, just like candidate channels feed fast channels.
+
+In this repository, the intended promotion flow is reflected by a `feeder` property in the channel declaration.
+For example, for [`channels/fast-4.2.yaml`](channels/fast-4.2.yaml):
+
+```yaml
+feeder:
+  name: candidate-4.2
+  delay: PT24H
+```
+
+which declares the intention that nodes and edges will be considered for promotion into `fast-4.2` after cooking for 24 hours in `candidate-4.2`.
+The `delay` value is an [ISO 8601][rfc-3339-p13] [duration][iso-8601-durations].
+
+This is the expected delay, but it does not mean that promotion will happen at that moment.
+For example, it is possible that release architects decide that there is insufficient data for a `fast-4.2` promotion, in which case the promotion can be delated until sufficient data accumulates.
+
+##### Tombstones
+
+Removing a node from a channel can strand existing clusters with a `VersionNotFound` error.
+To avoid that, unstable nodes are left in their existing channels, but should not be promoted to additional channels.
+This is reflected through entries in the optional `tombstones` property.
+For example, [`channels/candidate-4.2.yaml`](channels/candidate-4.2.yaml) has:
+
+```yaml
+tombstones:
```

Comment:
I propose naming it _frozen_ instead of _tombstones_.

Reply:
@steveeJ `frozen` does not communicate its intent. These are the nodes which can not have any future edge. So `tombstones` makes more sense.

Reply:
I think that _tombstones_ doesn't communicate it well either, since these versions are still in use somewhere, which is the reason why we need to give them an additional property. Also, It would be nice to stick to a more technical or neutral term here.

How about _immobilized_?

Reply:
I like tombstones more. These builds are similar to ghosts we don't speak of since they are dead to us and we don't want them coming back :)

Reply:
Tombstone has [an existing technical usage][1], which is similar to the usage here.

[1]: https://en.wikipedia.org/wiki/Tombstone_(data_store)